--- download-jdks.sh.old	2018-01-11 02:01:12.121144778 +0100
+++ download-jdks.sh	2018-01-11 02:02:20.607860902 +0100
@@ -92,7 +92,9 @@
     else # Linux
         7z x ../downloads/jdk-macosx-x64.dmg > /dev/null
         # The following seems dependent of the 7zip version. Travis on Version 9.20 extracts all partitions, where as at least version 16.02 is automatically extracting 4.hfs
-        7z x 4.hfs > /dev/null
+        if [ "$TRAVIS" != "SDK_DOCKER" ]; then
+            7z x 4.hfs > /dev/null
+        fi
         #install_xar
         #./xar-1.5.2/src/xar -xf JDK*/JDK*.pkg
         7z x JDK*/JDK*.pkg  > /dev/null
